runtime/native.go

```diff
@@ -348,6 +369,19 @@ func getNativeType(rtype reflect.Type) *Type {
 	return t
 }
 
+func newNativeField(name string, i, refDepth int) *Object {
+	// Closes over `i` and `refDepth`
+	nativeFieldGet := func(f *Frame, desc, instance *Object, owner *Type) (*Object, *BaseException) {
+		v := toNativeUnsafe(instance).value
+		for i := 0; i < refDepth; i++ {
+			v = v.Elem()
+		}
+		return WrapNative(f, v.Field(i))
+	}
+
+	return &Object{typ: &Type{slots: typeSlots{Get: &getSlot{nativeFieldGet}}}}
```

Comment:
It looks like args is always going to be the native instance because [propertyGet][0] always passes the instance argument, and the only instance we're putting this Get property on are native instances, right? I'm not seeing the scenario where this code could execute with a non-native instance...

[0]: https://github.com/google/grumpy/blob/master/runtime/descriptor.go#L65